scripts/utils.py

```diff
@@ -11,20 +11,14 @@ def read_json_file(file_name):
   
   return data
 
-def gsutil_cp(file, bucket, copy_to_remote=True, debug=True):
-  log_name = "gsutil_cp.log"
-  base_command = "gsutil -h 'Cache-Control:no-cache,max-age=0' cp -L {}".format(log_name)
+def gsutil_cp(file, bucket, copy_to_remote=True):
+  base_command = "gsutil -h 'Cache-Control:no-cache,max-age=0' cp"
```

Comment:
using verbose logging seemed to prevent the file from actually being copied... I'm removing it because it adds a lot of unnecessary noise, and was added for a specific bug we aren't encountering anymore. It wasn't helpful for debugging at the time either